### PR TITLE
Qa 2076 manage sets create automation

### PR DESCRIPTION
**Pull request by bpierce01**
## Description
- Added tests for create set in manage set
- Added tests for some alternative paths, text and link checking
- Added Automation IDs for container that holds the bottom row of buttons inside a modal for custom filters, creating sets, adding genes to existing sets, etc. 

The manage set tests work will be continued in QA-2077, so this is not a finished product

### CODE REVIEW

File: packages/portal-proto/src/components/ClearStoreErrorBoundary.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ class ClearStoreErrorBoundary extends Component<
             An unexpected error has occurred. Your saved cohorts are being
             recovered, but your cart and sets may be lost.
           </p>
-          <ModalButtonContainer>
+          <ModalButtonContainer data-testid="modal-button-container">
```
Comment (uc-pperumala) [inline packages/portal-proto/src/components/ClearStoreErrorBoundary.tsx:28]: If it's a button, this should start with `button-`

File: packages/portal-proto/src/components/InputEntityList/InputEntityList.tsx [hunk 2]
```diff
@@ -269,7 +269,7 @@ const InputEntityList: React.FC<InputEntityListProps> = ({
           />
         )}
       </div>
-      <ButtonContainer>
+      <ButtonContainer data-testid="modal-button-container">
```
Comment (uc-pperumala) [inline packages/portal-proto/src/components/InputEntityList/InputEntityList.tsx:272]: Same.
If it's a button, this should start with `button-`

**Review by uc-pperumala — APPROVED** [review]

File: .snyk [hunk 3]
```diff
@@ -0,0 +1,2 @@
+language-settings:
```
Comment (bpierce01) [inline .snyk:1]: Snyk uses python 3.7.4 by default, but that version of python does not support Pillow 10.0.1
So this .snyk file should update the version of python that Snyk uses to validate our dependencies 

Supporting document here: Document here: https://docs.snyk.io/scan-using-snyk/supported-languages-and-frameworks/python

File: packages/portal-proto/src/components/ClearStoreErrorBoundary.tsx (hunk 1 above)
Comment (bpierce01) [reply packages/portal-proto/src/components/ClearStoreErrorBoundary.tsx:28]: As discussed it is not a button, but rather the bottom portion of the modal that contains buttons

File: packages/portal-proto/src/components/InputEntityList/InputEntityList.tsx (hunk 2 above)
Comment (bpierce01) [reply packages/portal-proto/src/components/InputEntityList/InputEntityList.tsx:272]: As discussed it is not a button, but rather the bottom portion of the modal that contains buttons

File: .snyk (hunk 3 above)
Comment (bpierce01) [reply .snyk:1]: Had a conversation with the security team, and they said it is fine to try to see if it fixes the issue. We will know once it's merged in!

**Review by craigrbarnes — APPROVED** [review]

File: .snyk (hunk 3 above)
Comment (craigrbarnes) [reply .snyk:1]: I will merge now and we will see

**Pull request merged**